zanata-war/src/main/java/org/zanata/security/FacesSecurityEvents.java

```diff
@@ -34,6 +40,8 @@
  * Override the {@link org.jboss.seam.security.FacesSecurityEvents} component to
  * alter default values.
  *
+ * TODO [CDI] need to find a CDI replacement.
```

Comment:
On 2015-07-31 09:29, Patrick Huang wrote:
> In zanata-war/src/main/java/org/zanata/security/FacesSecurityEvents.java
> <https://github.com/zanata/zanata-server/pull/908#discussion_r35933312>:
> 
>> @@ -34,6 +40,8 @@
>>   * Override the {@link org.jboss.seam.security.FacesSecurityEvents} component to
>>   * alter default values.
>>   *
>> + * TODO [CDI] need to find a CDI replacement.
> 
> This class extends org.jboss.seam.security.FacesSecurityEvents. Need to
> do it in deltaspike or something else.

It doesn't extend anything in my checkout of this branch.  Does this
comment still apply?

Actually, I think the class is entirely unused now.


-- 
Sean Flanigan

Principal Software Engineer
Globalisation Tools Engineering
Red Hat